portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/Prototype/MockImplEndpoints.jsx

```diff
@@ -9,115 +9,767 @@
+                setNullScripts(prev => prev.filter(item => item !== `${method} - ${path}`));
+            }
+            forceUpdate();
+            Alert.info('Successfully Modified the mock script!');
+        } catch (e) {
+            Alert.error('Error generating mock scripts!');
+        } finally {
+            setAiLoadingStates(null); // Reset AI loading state
+            if (aiLoadingStates === null) {
+                setSaveDisable(false);
+            }
+        }
+
     }
 
-    return <>
-        <Grid container direction='row' justifyContent='flex-start' spacing={2} alignItems='stretch'>
-            <Grid item md={12}>
-                <Paper>
-                    {Object.keys(paths).map((path) => {
-                        return (
-                            <Grid key={path} item md={12}>
-                                <GroupOfOperations openAPI={swagger} tag={path}>
-                                    <Grid
-                                        container
-                                        direction='column'
-                                        justifyContent='flex-start'
-                                        spacing={1}
-                                        alignItems='stretch'
+    const handleGenerateScripts = async (useAI) => {
+        setProgress(true);
+        setSaveDisable(true);
+        try {
+            const payload = {
+                instructions: showInstructions ? mockConfig.config.instructions : undefined
+            }
+            const response = await generateMockScripts(useAI, payload)
+            if (useAI) updateMockDB({ [xWso2MockDBProperty]: response.obj[xWso2MockDBProperty] });
+            else updateMockDB({ [xWso2MockDBProperty]: undefined });
+            forceUpdate();
+            Alert.info('Successfully generated mock scripts!');
+            setMockConfig({ ...mockConfig, useAI })
+            setShowInstructions(false);
+            setIsFirstTimeSelection(false);
+            setSaveDisable(false);
+            setNullScripts([]);
+        } catch (e) {
+            console.error(e);
+            if (useAI) {
+                Alert.error('Error generating mock scripts with AI!. Please try again.')
+                setMockConfig({
+                    ...mockConfig, useAI: false
+                })
+            }
+            else {
+                Alert.error('Error generating mock scripts!');
+            }
+        } finally {
+            setProgress(false);
+        }
+    };
+
+    const handleConfigClick = (endpointConfig = {}) => {
+        const { path, method } = endpointConfig;
+        setCurrentConfig({ path, method })
+        setOpenConfig(true);
+    };
+
+    const authTokenNotProvidedWarning = (
+        <FormattedMessage
+            id='Apis.Details.Endpoints.Prototype.MockedOAS.warning.authTokenMissing'
+            defaultMessage={'You must provide a token to start using the AI-Assisted API Mock Server. To obtain one, '
```

Comment:
Let's be specific about the steps they need to follow for registration. In this case step 1 - 3. Once a documentation is added for this feature, we can point that link here.